prow/monitoring/mixins/Makefile

```diff
@@ -0,0 +1,20 @@
+#
+SHELL=/bin/bash -o pipefail
+
+all: tide.json
```

Comment:
I think we want all of these as `bazel` targets

Reply:
yes ... right ... let me give it a try.

Reply:
Oh ... yes ... i remember now.
I did not find a bazel rule for downloading the dependencies ... like `jb install`.
We could (as far as i know of couse)
* either push the dependencies as well and then [rules_jsonnet](https://github.com/bazelbuild/rules_jsonnet).
* or push the generated json files (this PR chose this way).

Reply:
I think the `rules_jsonnet` way would be best?

Reply:
> I think the `rules_jsonnet` way would be best?

Sure. Let me try.